sections/security/requestpayloadsizelimit.md

```diff
@@ -0,0 +1,60 @@
+# Limit payload size using a reverse-proxy or a middleware
+
+### One Paragraph Explainer
+
+When handling incoming requests in your web application, you should limit the size of their respective payloads. Incoming requests with
+unlimited body/payload sizes can lead to your application crashing due to a denial-of-service outage or other unwanted side-effects.
```

Comment:
@BrunoScheufler maybe throw a word or two about why this happens (handling large JSONs)

Reply:
@i0natan you're right, added a few details that body parsing, in general, requires a lot of performance, especially with larger payloads.